plugin.cpp

```diff
@@ -312,7 +326,7 @@ PLUGIN_HANDLE plugin_init(ConfigCategory* config)
 		
 		if ( ccAddress.size() != ccNames.size() )
 		{
-			Logger::getLogger()->error("There is mistach between cc address and cc names count");
+			Logger::getLogger()->error("There is mistach between cc address and cc names count, exiting...");
```

Comment:
Sane comment on exiting

Reply:
is it ok if I remove remove word "exiting" and keep rest of the message same?